index.html

```diff
@@ -5,47 +5,136 @@
     <meta name="description" content="lesson-9">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Natalia's page</title>
+    <link rel="stylesheet" href="css/index.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Manrope:wght@200..800&family=Montserrat:ital,wght@0,100..900;1,100..900&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
 </head>
 <body>
 
-
+<header>
   <nav>
-    <a href="#About">About</a> |
-    <a href="#Experience">Experience</a> |
-    <a href="#Connect">Connect</a>
+    <div class="nav-links">
+      <a href="index.html">HOME</a>
+      <a href="#about">ABOUT</a>
+      <a href="#experience">EXPERIENCE</a>
+      <a href="#projects">PROJECTS</a>
+      <a href="">BLOG</a>
+      <a href="#connect">CONTACT</a>
+    </div>
   </nav>
+</header>
+
+<main>
   
-    <h1>Natalia Sokolova</h1>
+   <div class = "content">
+      <div class = "contentsec">
+        <img src="img/IMG_1.jpg" alt="Pictute of web developer" width="400" height="300"> 
+      </div>
+      <div class = "contentsec2">
+        <h1>Hello! I'm Natalia Sokolova</h1>
+          <p>I am a freelance web developer specializing in high-quality Shopify & and WordPress development.</p>
+          <button type="button">Let's talk!</button> 
+      </div>      
+   </div>
```

Comment:
I don't think the class names `content` and `contentsec` are very descriptive of the styles applied to them nor their content. Consider naming those classes after what you're using them for. For instance, the `content` class only applies the property `display: flex`, so a better name would be `flex-container`.

Reply:
Updated.